itest/supply_commit_test.go

```diff
@@ -1199,6 +1199,38 @@ func testSupplyVerifyPeerNode(t *harnessTest) {
 		),
 	)
 
+	// If we query for the latest supply commitment from the universe server
```

Comment:
This itest is already rather long. I think we can improve the readability by moving this new assertion/logic into a helper assertion function. 

We can likely also spin out those assertions at the end into a helper function, as the same assertions are repeated many times above.